payment/src/main/java/org/killbill/billing/payment/core/janitor/IncompletePaymentTransactionTask.java

```diff
@@ -127,8 +127,8 @@ TransactionStatus updatePaymentAndTransactionIfNeeded(final UUID accountId,
                                                           final InternalTenantContext internalTenantContext) throws LockFailedException {
         // In the GET case, make sure we bail as early as possible (see PaymentRefresher)
         if (currentTransactionStatus != null && !TRANSACTION_STATUSES_TO_CONSIDER.contains(currentTransactionStatus)) {
-            // Nothing to do
-            return null;
```

Comment:
Shouldn't this be changed too?

https://github.com/killbill/killbill/blob/ce2ace3039250f4c87ac2afa16a51a498c045360/payment/src/main/java/org/killbill/billing/payment/core/janitor/IncompletePaymentTransactionTask.java#L159

Reply:
Yes, i think so too, will fix.